assets/scss/griidc-colors.scss

```diff
@@ -54,10 +54,10 @@ $gradientStart: #022f3a;
 $gradientEnd:   #4f6e74;
 
 $theme-colors: (
-    "primary":       $boston,
-    "secondary":     $blueBird,
+    "primary":       $dark-cyan,

```

Comment:
Is there a reason why we are changing the main template colors in this PR, because it would affect other apps as well

Reply:
Yes, these are GRIIDC colors, and they don't use Boston or Blue Bird, this was a bad copy and paste from the NAS scss.

Reply:
I think the POs didn't want these colors on Search app and Research group landing page apps, Changing these would change the colors on them.

Reply:
I see now, that was maybe maybe not the best thing to do, because I already got confused.